src/control/lib/support/log.go

```diff
@@ -579,6 +659,90 @@ func copyServerConfig(log logging.Logger, opts ...CollectLogsParams) error {
 	return nil
 }
 
+// Copy only specific lines from the server logs based on the Start/End date
+// and Start/End time, provided by user.
+func cpLinesFromLog(log logging.Logger, srcFile string, destFile string, opts ...CollectLogsParams) error {
+	// Copy the full log file in case of no dates provided
+	if opts[0].LogStartDate == "" && opts[0].LogEndDate == "" {
+		err := cpLogFile(srcFile, destFile, log)
+		if err != nil {
+			return err
+		}
+	} else {
```

Comment:
Could return nil for this case, then the nested "else" would not be necessary.

Reply:
I will update the return code as nil.

Reply:
ok I will update this.